packages/wellknown-did-issuer/src/agent/WellKnownDidIssuer.ts

```diff
@@ -0,0 +1,239 @@
+import { parseDid } from '@sphereon/ssi-sdk-core'
+import {
+  DomainLinkageCredential,
+  IDidConfigurationResource,
+  IssuanceCallback,
+  ServiceTypesEnum,
+  WellKnownDidIssuer as Issuer,
+} from '@sphereon/wellknown-dids-client'
+import { IAgentPlugin, IIdentifier, VerifiableCredential } from '@veramo/core'
+import { OrPromise } from '@veramo/utils'
+import { normalizeCredential } from 'did-jwt-vc'
+import { Service } from 'did-resolver/lib/resolver'
+import { Connection } from 'typeorm'
+import { DidConfigurationResourceEntity, createCredentialEntity, didConfigurationResourceFrom } from '../entities/DidConfigurationResourceEntity'
+import {
+  IAddLinkedDomainsServiceArgs,
+  IWellKnownDidIssuer,
+  IWellKnownDidIssuerOptionsArgs,
+  IRegisterIssueCredentialArgs,
+  IRemoveCredentialIssuanceArgs,
+  RequiredContext,
+  IIssueDidConfigurationResourceArgs,
+  IIssueDomainLinkageCredentialArgs,
+  IGetDidConfigurationResourceArgs,
+  ISaveDidConfigurationResourceArgs,
+} from '../types/IWellKnownDidIssuer'
+import { schema } from '../index'
+
+/**
+ * {@inheritDoc IWellKnownDidIssuer}
+ */
+export class WellKnownDidIssuer implements IAgentPlugin {
+  readonly schema = schema.IWellKnownDidVerifier
+  readonly methods: IWellKnownDidIssuer = {
+    addLinkedDomainsService: this.addLinkedDomainsService.bind(this),
+    getDidConfigurationResource: this.getDidConfigurationResource.bind(this),
+    issueDidConfigurationResource: this.issueDidConfigurationResource.bind(this),
+    issueDomainLinkageCredential: this.issueDomainLinkageCredential.bind(this),
+    registerCredentialIssuance: this.registerCredentialIssuance.bind(this),
+    removeCredentialIssuance: this.removeCredentialIssuance.bind(this),
+    saveDidConfigurationResource: this.saveDidConfigurationResource.bind(this),
+  }
+
+  private readonly credentialIssuances: Record<string, IssuanceCallback>
+  private readonly didConfigurationResourceRelations = ['linkedDids']
+
+  constructor(private dbConnection: OrPromise<Connection>, args?: IWellKnownDidIssuerOptionsArgs) {
+    this.credentialIssuances = (args && args.credentialIssuances) || {}
```

Comment:
Interesting. Is this working? My first reaction is that the first part of the expression results in a boolean instead of assigning the credentialIssuances object

Reply:
it should. i removed the brackets and double tested it